### PR TITLE
[v3] updating docs with new license management flow and UX

### PR DESCRIPTION
**Pull request by SimonWallner**
- reflecting the upcoming changes in license and user management.
- update most references to API_KEY to auth token
- update most references to 'API keys' in ci context to 'CI key'
- updated links into web app.

# Merge when v3 is released!
don't merge before rolling out changes in front-end https://github.com/localstack/localstack-web/pull/1001

Todo
- [x] add missing screenshots, update screenshots where needed
- [x] check all links into the platform and update to new paths

### CODE REVIEW

File: content/en/getting-started/auth-token.md [hunk 1]
```diff
@@ -0,0 +1,123 @@
+---
+title: "Auth Token"
+weight: 20
+categories: ["LocalStack Pro & Enterprise"]
+description: >
+  Configure your auth token to start LocalStack
+aliases:
+  - /get-started/pro/
```
Comment (HarshCasper) [inline content/en/getting-started/auth-token.md:8]: There is no need for an alias now

File: content/en/getting-started/auth-token.md [hunk 2]
```diff
@@ -0,0 +1,123 @@
+---
+title: "Auth Token"
+weight: 20
+categories: ["LocalStack Pro & Enterprise"]
```
Comment (HarshCasper) [inline content/en/getting-started/auth-token.md:4]: No need for this either

File: content/en/user-guide/web-application/users-licences/index.md [hunk 3]
```diff
@@ -0,0 +1,73 @@
+---
+title: "Managing Users and Licenses"
+tags: ["LocalStack Pro", "Accounts"]
```
Comment (HarshCasper) [inline content/en/user-guide/web-application/users-licences/index.md:3]: We can remove the tags. Please add a `linkTitle` here

File: content/en/user-guide/web-application/workspaces/index.md [hunk 4]
```diff
@@ -0,0 +1,15 @@
+---
+title: "Workspaces"
+tags: ["LocalStack Pro", "Accounts"]
+weight: 20
+description: >
+  A workspace is the base organizational unit in the LocalStack web app.
```
Comment (HarshCasper) [inline content/en/user-guide/web-application/workspaces/index.md:6]: `LocalStack Web Application`. Let's keep constancy in the terminology

File: content/en/user-guide/web-application/teams/index.md [hunk 5]
```diff
@@ -1,27 +0,0 @@
----
-title: "Teams"
-tags: ["LocalStack Pro"]
-weight: 2
-description: >
-  LocalStack Organizations/Teams offer a flexible way of managing access roles, permissions, and API keys for your team members.
----
-
```
Comment (HarshCasper) [inline content/en/user-guide/web-application/teams/index.md:8]: I would prefer to not delete this page for now, and rather put up a deprecation notice. WDYT?

File: content/en/user-guide/ci/circle-ci/index.md [hunk 6]
```diff
@@ -64,9 +64,9 @@ The above CircleCI job does the following:
 
 ## Configuring a CI key
 
-To enable LocalStack Pro+, you need to add your LocalStack CI API key to the project's environment variables. The LocalStack container will automatically pick it up and activate the licensed features. 
+To enable LocalStack Pro+, you need to add your LocalStack CI API key to the project's environment variables. The LocalStack container will automatically pick it up and activate the licensed features.
 
-Go to the [API Key Settings](https://app.localstack.cloud/account/apikeys) page and copy your CI key. To add the CI key to your CircleCI project, follow these steps:
+Go to the [CI Key PAge](https://app.localstack.cloud/workspace/ci-keys) page and copy your CI key. To add the CI key to your CircleCI project, follow these steps:
```
Comment (HarshCasper) [inline content/en/user-guide/ci/circle-ci/index.md:69]: `CI Key Page`

**Review by HarshCasper — CHANGES_REQUESTED** [review]
Few more comments: 

- The terminology is a bit unclear. Is it `Auth Token` or `auth token`? I guess we should go with the former consistently. 
- Rather than deleting the pages (which might trouble other users), let's put up a deprecation notice for the time being.
- The docs are unclear about the path for API keys. Have they been deprecated? What's the timeline wherein they still work? And more questions.

File: content/en/getting-started/auth-token.md (hunk 1 above)
Comment (SimonWallner) [reply content/en/getting-started/auth-token.md:8]: probably some copy paste error

File: content/en/user-guide/web-application/teams/index.md (hunk 5 above)
Comment (SimonWallner) [reply content/en/user-guide/web-application/teams/index.md:8]: not sure. I think the docs should only provide 1 way of doing things. there is no need to keep the old info around when it comes to license management

Comment (HarshCasper) [reply content/en/user-guide/web-application/teams/index.md:8]: I understand the point, but this won't be beneficial for existing users who have not been notified about this and would be confused about this. We have traditionally kept deprecated docs around (Cockpit vs Desktop app as an example), and removed them once we are sure that everyone has been notified of the change

Comment (SimonWallner) [reply content/en/user-guide/web-application/teams/index.md:8]: this page was made obsolete by the 'users and licences 'page

**Review by HarshCasper — APPROVED** [review]

**Pull request merged**